data/governments.txt

```diff
@@ -398,3 +398,9 @@ government "Wanderer"
 	"friendly disabled hail" "wanderer untranslated"
 	"hostile hail" "wanderer untranslated"
 	"hostile disabled hail" "wanderer untranslated"
+
+government "Wormhole Alpha"
```

Comment:
Perhaps add a comment here?
```suggestion
# A dummy government used to prevent Hai from entering the wormhole in Ultima Thule, but allowing human merchants to enter.
government "Wormhole Alpha"
```

I just realized though that this is going to cause an issue with "Pirate Troubles [4]" where the player needs to escort a number of Hai Geocoris out of Hai space and back.